puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_proxy_develop_pr_rubocop.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -ex
+
+APP_ROOT=`pwd`
+
+# setup basic settings file
+cp $APP_ROOT/config/settings.yml.example $APP_ROOT/config/settings.yml
+
+# RVM Ruby environment
+. /etc/profile.d/rvm.sh
+# Use a gemset unique to each executor to enable parallel builds
+gemset=$(echo ${JOB_NAME} | cut -d/ -f1)-${EXECUTOR_NUMBER}
+rvm use ruby-${ruby}@${gemset} --create
+rvm gemset empty --force
+# Update any gems from the global gemset
+gem update --no-ri --no-rdoc
+gem install bundler --no-ri --no-rdoc
+
+bundle install --with=test '--without=development puppet windows bmc' --retry 5
```

Comment:
Are those quotes needed? AFAIK it should work without them.

Reply:
bundle docs describe `--without` taking a space separated list, so i think the quotes are needed otherwise bash will treat each as a separate token.